test/e2e/e2e_suite_test.go

```diff
@@ -37,7 +37,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1beta1"
-	"sigs.k8s.io/cluster-api-provider-vsphere/test/helpers"
+	helpers "sigs.k8s.io/cluster-api-provider-vsphere/test/framework"
```

Comment:
Importing framework as helpers is a bit confusing to me. Wondering if we can maybe use vsphereframework or testframework or something (in some other files we have to avoid the conflict with "framework" from core capi)

(same in other files)

Reply:
I can also take this as a follow-up